crescent/commands/options.py

```diff
@@ -152,30 +154,47 @@ def option(
     ...
 
 
+# fmt: off
```

Comment:
Looks like you only disable formatting around the overrides that would be formatted. For the sake of consistency, I think we should just disable formatting before the first override, and then re-enable after the last override. Also add a comment explaining why.

Reply:
Override? Overload?

Reply:
Sorry, overload lol.

Reply:
Doing this though kind of turns off formatting for all of them...

Reply:
I think its better how it is right now

Reply:
After some testing it appears that black won't format a function if there's a comment after one of its arguments. I don't think we even need to disable formatting at all here.

Reply:
Nevermind, that's because I had magic trailing comma enabled.

Reply:
Resolved in discord.